examples/OpenCLVectorAdd/OpenCLVectorAdd.cpp

```diff
@@ -123,21 +124,24 @@ int main()
             ZoneScopedN("Write Buffer A");
             TracyCLZoneS(tracyCLCtx, "Write BufferA", 5);
 
-            CL_ASSERT(clEnqueueWriteBuffer(commandQueue, bufferA, CL_TRUE, 0, N * sizeof(float), hostA.data(), 0, nullptr, &writeBufferAEvent));
+            CL_ASSERT(clEnqueueWriteBuffer(commandQueue, bufferA, CL_FALSE, 0, N * sizeof(float), hostA.data(), 0, nullptr, &writeBufferAEvent));
 
             TracyCLZoneSetEvent(writeBufferAEvent);
         }
         {
             ZoneScopedN("Write Buffer B");
             TracyCLZone(tracyCLCtx, "Write BufferB");
 
-            CL_ASSERT(clEnqueueWriteBuffer(commandQueue, bufferB, CL_TRUE, 0, N * sizeof(float), hostB.data(), 0, nullptr, &writeBufferBEvent));
+            CL_ASSERT(clEnqueueWriteBuffer(commandQueue, bufferB, CL_FALSE, 0, N * sizeof(float), hostB.data(), 0, nullptr, &writeBufferBEvent));
 
             TracyCLZoneSetEvent(writeBufferBEvent);
         }
     }
 
-    for (int i = 0; i < 10; ++i)
+    const int num_launchs = 100;
```

Comment:
Can I suggest that you keep the variables names consistent with the rest of the example, i.e. camelCase?